### PR TITLE
Components

### PR DESCRIPTION
**Pull request by Patfarmurs**

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+          <button type="submit" onClick={(event) => handleClick(event, state, setter)}>2</button>
+          <button type="submit" onClick={(event) => handleClick(event, state, setter)}>3</button>
+          <button type="submit" onClick={(event) => handleClick(event, state, setter)} className="box-color">+</button>
+          <button type="submit" onClick={(event) => handleClick(event, state, setter)} className="zero">0</button>
+          <button type="submit" onClick={(event) => handleClick(event, state, setter)}>.</button>
+          <button type="submit" onClick={(event) => handleClick(event, state, setter)} className="box-color">=</button>
+        </div>
+      </div>
+    </section>
+  );
```
Comment (Lucas-Masaba) [inline src/components/Calculator.js:45]: - [ ] Please design your calculator to match the design provided.
### Your implementation
image
### Required
image

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @Patfarmurs  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Linter checks passing ✔️ 
- Successfully implemented most requirements 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [ ] Please delete the `App.test.js` file in your project(The tests do not match the current project and would fail.). You are required to remove the boilerplate template and this file is one of them.

- [ ] In your `README.md`, please customise your feature features header to suit your project. Currently, it has default values
image

File: src/components/Calculator.js (hunk 1 above)
Comment (Patfarmurs) [reply src/components/Calculator.js:45]: Thanks, @Lucas-Masaba for your feedback.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+          <button type="submit" onClick={(event) => handleClick(event, state, setter)} className="box-color">x</button>
+          <button type="submit" onClick={(event) => handleClick(event, state, setter)}>4</button>
+          <button type="submit" onClick={(event) => handleClick(event, state, setter)}>5</button>
+          <button type="submit" onClick={(event) => handleClick(event, state, setter)}>6</button>
+          <button type="submit" onClick={(event) => handleClick(event, state, setter)} className="box-color">-</button>
+          <button type="submit" onClick={(event) => handleClick(event, state, setter)}>1</button>
+          <button type="submit" onClick={(event) => handleClick(event, state, setter)}>2</button>
+          <button type="submit" onClick={(event) => handleClick(event, state, setter)}>3</button>
+          <button type="submit" onClick={(event) => handleClick(event, state, setter)} className="box-color">+</button>
+          <button type="submit" onClick={(event) => handleClick(event, state, setter)} className="zero">0</button>
```
Comment (franclobo) [inline src/components/Calculator.js:39]: - [ ] Well done implementing the changes requested by the last code reviewer, but take in mind the zero on the display is not a button, please fix it for a better user experience 👇🏽 

image

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Patfarmurs, 

# ♻️ Required Changes ♻️

soClose

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors ✔️ 
- Correct git flow ✔️ 
- Professional README file ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 2 above)
Comment (Patfarmurs) [reply src/components/Calculator.js:39]: @franclobo I did not understand your comment, would you please be more clear.

**Review by KayLemba — APPROVED** [review]
[# STATUS: APPROVED :green_circle: :clap: 

Hello  @Patfarmurs        , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: aa
Congratulations! 🎉 

HIGHLIGHTS:

- [x]  Your repo is professional 
- [x] Descriptive P.R
- [x] Nice UI Design







 








Now you can merge this branch and move on :1st_place_medal: 
Happy Coding cheers 🥇 💯 



well-1

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._]

**Pull request merged**